src/WebApi/Domain/Models/ValueObject/Category.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Domain.Models.ValueObject;
+
+public class Category
```

Comment:
@petrspelos what do you think about category vs tags? We should probably pick the right naming ASAP. To be fair I mentioned tags to @TerezaSkutova so it's not her fault 😄

Reply:
My thinking is the following:

- Category seems like one per question
- Tag indicate you could have as many of them on one question as you want

So if our question can only have one of these, Category would make sense to me.

@PlesnikJakub or can we assign multiple of these to a question? 🤔

Reply:
Actually if we were to change the domain so question can have multiple tags, then that would make an exemplary task and we could have much better question categorization, since our current categories are mostly focused on programming languages and I can imagine a use case where someone wants a question with tags like `Tree` `Depth First Search`.

Reply:
I can imagine that it would work much better with tags to be more specific for question categorization. 
You should be able to know what is the target of the question without reading it.

What does EF core context mean? -  [ .NET, EF ]
What is default implementation in an interface for? [ OOP, .NET, JAVA]  
Can you use the interface in python? [ OOP, Python ]

Reply:
If we extended the model by color, it would be more tag not category. From that point of view it doesn't make sense to set color for a category object.

Reply:
I would go forward with the solution as is but we need to immediately address this and I propose to fully switch to naming as Tags.  We do have other naming issues on the infrastructure level so I am wondering if this should be one db-focused task that we can define.